assets/scss/6-components/button/_button.scss

```diff
@@ -89,9 +89,11 @@
     }
   }
 
-  &--compact {
+  &--standard {
```

Comment:
Unless I'm missing some use cases, it seems this style is used primarily on anchors, not buttons. Should the naming reflect that? Maybe it'll be too confusing since we also have a [style](https://ttds.surge.sh/pages/components/index.html#link-button-c-link-button) for the opposite: buttons styled to look like anchors. IDK.

Reply:
Yeah, that's an interesting thought. I guess in my head I don't think a style distinguishes between buttons and anchors. It matters what you use semantically, but the `c-button` class just says I want whatever this is to look like our button style. I think naming it something about it being a link unnecessarily narrows the scope of where it can be used.

I do see your point though in that a name could offer guidance of when we use it. I just hate to create a limitation because of the off chance that a new use of it does find its way into a design.

Reply:
Makes sense ... my one counterpoint would be that, in the case of [this style](https://ttds.surge.sh/pages/components/index.html#link-button-c-link-button), the element it was used on _did_ matter. That's what caused me to create it in the first place: Our normal anchor styles weren't enough to make a button look like a link.

I'm sure it's fine; just noting.

Reply:
Totally and I meant to mention that the need for that c-link-button def makes sense because `t-links` does [that little number](https://github.com/texastribune/queso-ui/blob/master/assets/scss/5-typography/_links.scss#L14-L21) with inheritance. I don't love that t-links class because it's not direct, but it sure has been handy to stick on top-level elements.